src/test/java/com/revature/controllers/ProfileControllerTest.java

```diff
@@ -5,89 +5,100 @@
  *         notation using Junit and EasyMock?"
  *         https://stackoverflow.com/questions/9807882/unit-testing-a-class-with-autowired-notation-using-junit-and-easymock
  */
-/*
- * 
- * package com.revature.controllers;
- * 
- * import static org.junit.Assert.assertEquals; import static
- * org.junit.Assert.assertTrue;
- * 
- * import java.util.ArrayList; import java.util.List;
- * 
- * import org.junit.Before; import org.junit.Test; import org.mockito.Mockito;
- * import org.springframework.test.util.ReflectionTestUtils;
- * 
- * import com.revature.controller.ProfileController; import
- * com.revature.model.Profile; import com.revature.service.ProfileService;
- * 
- * public class ProfileControllerTest { ProfileController trueController;
- * ProfileController falseController; ProfileService trueService; ProfileService
- * falseService; Profile profileOne; Profile profileTwo;
- * 
- * @Before
- *//**
-	 * Creates two instances each of the profile, service, and controller classes,
-	 * using Mockito mock instances in the case of service.
-	 */
-/*
- * public void setup() { trueService = Mockito.mock(ProfileService.class);
- * falseService = Mockito.mock(ProfileService.class); profileOne = new
- * Profile(1, "Carrie", "Jones", "This is a test profile."); profileTwo = new
- * Profile(2, "Oscar", "Smith", "This is another test profile."); trueController
- * = new ProfileController(); falseController = new ProfileController();
- * ReflectionTestUtils.setField(trueController, "ps", trueService);
- * ReflectionTestUtils.setField(falseController, "ps", falseService);
- * Mockito.when(trueService.insertProfileInfo(profileOne)).thenReturn(true);
- * Mockito.when(trueService.insertProfileInfo(profileTwo)).thenReturn(true);
- * Mockito.when(falseService.insertProfileInfo(profileOne)).thenReturn(false);
- * Mockito.when(falseService.insertProfileInfo(profileTwo)).thenReturn(false);
- * List<Profile> listOfTwo = new ArrayList<>(); listOfTwo.add(profileOne);
- * listOfTwo.add(profileTwo); List<Profile> emptyList = new ArrayList<>();
- * Mockito.when(trueService.getAllProfiles()).thenReturn(listOfTwo);
- * Mockito.when(falseService.getAllProfiles()).thenReturn(emptyList); }
- * 
- * @Test
- *//**
+  package com.revature.controllers;
```

Comment:
It looks like when this file was uncommented, the formatting was messed up. Can you go in and clean up the formatting, such as tabbing the member variables forward, and having a blank line between the package statement and the import statements, alignment of curly braces and comments, etc. Additionally, go ahead and remove any commented out code if it is left.

Reply:
reformatted